RELEASE.md

```diff
@@ -358,3 +358,100 @@ actually push artifacts to the real release channels.
 Also note that there is no additional automation around this
 currently. If you want to initiate a fake release, you must take that
 action yourself.
+
+# "Yanking" a Release
+
+In general, if an escaped defect is discovered in a Habitat release,
+the first solution should be to "roll forward". That is, a patch that
+addresses the defect (either by fixing it or reverting the code
+responsible) should be merged to master, and then a new release is
+built from that. This is the desired solution because it requires no
+extraordinary measures by end-users to take advantage of the fix; they
+only need to update their software as they would for any "normal"
+release.
+
+In the rare case that that is _not_ possible (e.g., the cause of the
+defect is not readily apparent), you can remove or "yank" the
+offending release, and fall back to the previous release. This is not
+a perfect solution, however. The process and drawbacks are detailed
+here.
```

Comment:
It might be worth emphasizing here that we have a very **strong** preference for rolling forward and yanking a release should be considered a measure of last resort, after everything else has been exhausted.